osu.Game/Graphics/UserInterfaceV2/FormSliderBar.cs

```diff
@@ -97,9 +99,31 @@ public float KeyboardStep
             }
         }
 
+        /// <summary>
+        /// Whether to format the tooltip as a percentage or the actual value.
+        /// </summary>
+        public bool DisplayAsPercentage { get; init; }
+
+        /// <summary>
+        /// Whether sound effects should play when adjusting this slider.
+        /// </summary>
+        public bool PlaySamplesOnAdjust { get; init; }
+
+        /// <summary>
+        /// The string formatting function to use for the value label.
+        /// </summary>
+        public Func<T, LocalisableString> LabelFormat { get; init; }
+
+        /// <summary>
+        /// The string formatting function to use for the slider's tooltip text.
+        /// If not provided, <see cref="LabelFormat"/> is used.
+        /// </summary>
+        public Func<T, LocalisableString> TooltipFormat { get; init; }
```

Comment:
Why do both of these exist? Is this degree of control necessary?

Reply:
Consider `AudioOffsetAdjustControl`'s slider bar, I would expect the label to show `XXX ms` and the tooltip `XXX ms (hit objects appear later)`.